neurom/io/datawrapper.py

```diff
@@ -36,7 +36,25 @@ class DataWrapper(object):
     '''Class holding a raw data block and section information'''
 
     def __init__(self, data_block, fmt, sections=None):
-        '''Section Data Wrapper'''
+        '''Section Data Wrapper
+
+        data_block is np.array-like with the following columns:
+            [X, Y, Z, R, TYPE, ID, P]
+            X(float): x-coordinate
+            Y(float): y-coordinate
+            Z(float): z-coordinate
+            R(float): radius
+            TYPE(integer): one of the types described by POINT_TYPE
+            ID(integer): unique integer given to each point, the `ROOD_ID` is -1
+            P(integer): the ID of the parent
+
+        Notes:
+            - there is no ordering constraint: a child can reference a parent ID that comes
```

Comment:
@juanchopanza: please edit this, add to it if there are other invariants, or if I got something wrong